options/options.js

```diff
@@ -227,11 +229,16 @@ function removeEventListeners() {
 
 function prepareGroupEditor(event) {
   groupEditor.adjustUrlsColumnWidths();
+  groupEditor.resetSearch();
 
   if (event.relatedTarget == document.getElementById('addGroup')) {
     groupEditor.clear();
   }
 }
+function closeGroupEditor(event) {
+    groupEditor.resetSearch();
+    $('#groupEditorModal').modal('hide');
+}
```

Comment:
Add a blank line before the function and adjust indentation to 2 spaces instead of 4.